snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: plexmediaserver
-version: 1.2.2.2857-d34b464-1
+version: 1.2.2.2857-d34b464
```

Comment:
@sergiusens so what is your approach then to track snap content to git state relationship?